src/icons/svgs.test.js

```diff
@@ -0,0 +1,42 @@
+const path = require("path");
+const fs = require("fs");
+
+describe("svgs", () => {
```

Comment:
These tests ensure that svgs we add in the future follow the rules for making the svg color and size configurable